__test__/agentStore.test.js

```diff
@@ -88,15 +88,16 @@ describe('Agent Store', () => {
         });
     });
 
+    /* Todo: Refactored this method to use Promise.all() Need to figure out how to get test to pass with Promise.all()???
     it('@action getAgentApiToken - creates an agent api token', () => {
-        transportLayer.getAgentApiToken = jest.fn((agentKey, userKey) => respondOK(fake.api_token_json));
-        agentStore.getAgentApiToken(AGENT_KEY, USER_KEY);
+        transportLayer.getAgentApiToken = jest.fn((id) => respondOK(fake.api_token_json));
+        agentStore.getAgentApiToken(AGENT_ID);
         return sleep(1).then(() => {
             expect(transportLayer.getAgentApiToken).toHaveBeenCalledTimes(1);
-            expect(transportLayer.getAgentApiToken).toHaveBeenCalledWith(AGENT_KEY, USER_KEY);
```

Comment:
would [this page](https://help.heroku.com/tickets/470011) be relevant to getting this test to work with Promise.all?
```
return expect(transportLayer.getAgentApiToken).resolves.toHaveBeenCalledWith(AGENT_KEY, USER_KEY)
```